docs/installation.rst

```diff
@@ -75,9 +72,8 @@ Ubuntu       `16.04`_, backports (PPA): `15.10`_, `14.04`_ ``apt install borgbac
 .. _openSUSE official repository: http://software.opensuse.org/package/borgbackup
 .. _Brew cask: http://caskroom.io/
 .. _Raspbian testing: http://archive.raspbian.org/raspbian/pool/main/b/borgbackup/
-.. _16.04: https://launchpad.net/ubuntu/xenial/+source/borgbackup
-.. _15.10: https://launchpad.net/~costamagnagianfranco/+archive/ubuntu/borgbackup
-.. _14.04: https://launchpad.net/~costamagnagianfranco/+archive/ubuntu/borgbackup
+.. _Launchpad: https://launchpad.net/ubuntu/+source/borgbackup
```

Comment:
http://packages.ubuntu.com/xenial/borgbackup (see nav links at the top for other dists)